src/main/java/by/custom/utilcalculator/exception/UtilsborException.java

```diff
@@ -0,0 +1,20 @@
+package by.custom.utilcalculator.exception;
+
+import org.telegram.telegrambots.meta.api.objects.Message;
+
+public abstract class UtilsborException extends Exception {
+    public static final String USER_FILE_NOT_FOUND = "user_file_not_found";
+    public static final String READING_FROM_FILE_EXCEPTION = "file_reading_error";
+    public static final String SAVING_INTO_FILE_EXCEPTION = "file_saving_error";
+    public abstract String getExceptionCode();
+
+    public abstract String getFileNameExceptionDetails(final String filename);
```

Comment:
What if exception is not connected with file. Database exception for example. I think this method is not needed

Reply:
Remoced.